lldb/test/API/functionalities/process_save_core_minidump/TestProcessSaveCoreMinidumpYaml.py

```diff
@@ -0,0 +1,142 @@
+"""
+Test saving a mini dump, from yamilized examples.
+"""
+
+import os
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class ProcessSaveCoreMinidumpTestCaseYaml(TestBase):
+    def process_from_yaml(self, yaml_file):
+        minidump_path = self.getBuildArtifact(os.path.basename(yaml_file) + ".dmp")
+        self.yaml2obj(yaml_file, minidump_path)
+        self.target = self.dbg.CreateTarget(None)
+        self.process = self.target.LoadCore(minidump_path)
+        return self.process
+
+    def test_saving_sub_memory_range(self):
+        """
+        Validate we can save a Minidump for a subsection of a memory range.
+        I.E.
+        If our memory range is 0x1000-0x2000 nd the user specifies 0x1200-0x1800
```

Comment:
These examples are nice and easy to understand, but the YAML regions are defined with "hard" addresses like `0x7FFF12A84030`. Any reason we can't make the ranges in the YAML def easier to reason about?

Reply:
Gave this a shot (looks more like decimals), let me know you what you think @dmpots!

Reply:
Yes, looks easier to understand. I think it would be even better to match up the comments with the actual ranges. So either make the ranges match or change the comment to the actual thing being tested, e.g. 


> If our memory range is 0x1000-0x1100 and the user specifies 0x1000-0x1008 we should still capture 0x1000 to 0x1008

Also, typo "nd" -> "and"